conformance/proto/grpcechoserver/grpcecho.proto

```diff
@@ -0,0 +1,93 @@
+// Copyright 2023 The Kubernetes Authors.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
```

Comment:
so the server lives in https://github.com/kubernetes-sigs/ingress-controller-conformance and the client will live here ?

Reply:
That is currently correct. However, I received some feedback from @robscott that that image should live in this repo, so I will soon merge that PR into this one.

Reply:
lets mark this PR as draft until that done ? or do you want to introduce it in a follow up PR ?

Reply:
I just synced with @robscott on this. It sounds like we want to include that image in this PR despite the fact that it will become quite large. Marking this as draft until that happens (hopefully later today).